Sources/AST/ASTVisitor/ASTVisitor.swift

```diff
@@ -983,10 +983,14 @@ public struct ASTVisitor {
 
   func visit(_ functionArgument: FunctionArgument, passContext: ASTPassContext) -> ASTPassResult<FunctionArgument> {
     var processResult = pass.process(functionArgument: functionArgument, passContext: passContext)
+
+    processResult.passContext.isFunctionCallArgumentLabelContext = true
```

Comment:
Does setting a simple boolean work with nested function calls? What if I call another function and label the resultant value in a function call?

e.g. `myFun(abc: def(param: 1))`

Reply:
the reason I ask is since this is set on the pass context – don't we have a separate function call context where this could be stored instead or am I wrongly thinking about this?

Reply:
I believe this should not be a problem. The flag is set to true if and only if we are visiting an identifier for an argument label (abc and param above), which is different form the identifier expression (or an expression containing an identifier, such as def). I am going to add this to a test though, to be sure.

Reply:
It could also probably be set on some other context such as a function context indeed, but I don't think it would make a difference.

Reply:
If it's not the function context, doesn't it get passed around even after set? i.e. anything the AST  following the identifier and below the function call will have this flag set to true, right? 

I think adding a test is a good idea just to make sure 👍

Reply:
But it's being set to false right after visiting the identifier, right? Shouldn't that do it?

Reply:
And the identifier is a terminal node in the AST (at least for now)